test/integration/utils.test.js

```diff
@@ -160,7 +160,7 @@ describe(Support.getTestDialectTeaser('Utils'), () => {
       ]);
     });
 
-    if (Support.getTestDialect() !== 'mssql') {
+    if (Support.getTestDialect() !== 'mssql' && Support.getTestDialect() !== 'oracle') {
```

Comment:
What is the reason for disabling these tests? (I also don't see why mssql is disabling it)

Reply:
Conditions like CAST(A > 1) needs conversion from BOOLEN to NUMBER which is not present in Oracle. It might need few more change which needs further exploration.